jkube-kit/doc/src/main/asciidoc/inc/generator/_options_common.adoc

```diff
@@ -49,6 +49,11 @@ or already added by a generator which has been run previously.
 | *tags*
 | A comma separated list of additional tags you want to tag your image with
 | `jkube.generator.tags`
+
+| *buildpacksBuilderImage*
+|  Configure https://buildpacks.io/docs/for-platform-operators/concepts/builder/[BuildPack builder] OCI image for BuildPack Build. This field is applicable only in `buildpacks` build strategy.
+Defaults to `null`
```

Comment:
Are we sure this defaults to null?

Reply:
This is BaseGenerator's config option. I see we've put `null` as default value there. https://github.com/rohanKanojia/jkube/blob/pr/issue2470/jkube-kit/generator/api/src/main/java/org/eclipse/jkube/generator/api/support/BaseGenerator.java#L81

Reply:
Yes, that's code-wise. However, from a user's perspective, the default value is not null, but whatever default image we're using.

Reply:
Shall I add `paketobuildpacks/builder:base` here instead?

Reply:
I think that would make more sense than `null`